vmdk_plugin/plugin.go

```diff
@@ -209,6 +221,10 @@ func (d *vmdkDriver) unmountVolume(name string) error {
 // Name and driver specific options passed through to the ESX host
 func (d *vmdkDriver) Create(r volume.Request) volume.Response {
 
+	// Lock this volume for create
+	d.lockVolume(r.Name)
+	defer d.unlockVolume(r.Name)
```

Comment:
@msterin 
The remove case was a bad example.
As much as the daemon seems to be protecting us to start a container with a volume in the middle of Create() i would like to keep the locks there just to be safe.